src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php

```diff
@@ -81,7 +81,7 @@ public function process(ContainerBuilder $container)
 
                 if (!isset($event['method'])) {
                     $event['method'] = 'on'.preg_replace_callback([
-                        '/(?<=\b)[a-z]/i',
+                        '/(?<=\b|_)[a-z]/i',
                         '/[^a-z0-9]/i',
```

Comment:
btw this 2nd regex is a leftover from https://github.com/symfony/symfony/pull/1162/files it seems, where it was replaced with empty string which now happens at L87, this happened with refactoring likely.

AFAIK it's redundant :angel:

Reply:
it confused me too, but I'm not sure we can remove